flask_restx_patched/swagger.py

```diff
@@ -21,3 +21,20 @@ def parameters_for(self, doc):
         else:
             default_location = 'query'
         return schema2parameters(schema, default_in=default_location, required=True)
+
+    def expected_params(self, doc):
+        if 'params' in doc.keys():
+            params = doc.get('params', None)
+            assert params is not None
+
+            for name, param in params.items():
+                get_func = getattr(param, 'get', None)
+                if get_func is None:
+
+                    def _patched_get(value, default):
+                        return param.context.get(value, default)
+
+                    param.get = _patched_get
```

Comment:
What's this patching about?

Reply:
The error was tracked down to [here](https://github.com/python-restx/flask-restx/blob/master/flask_restx/swagger.py#L350).  There is a call a few lines above on line 343 that I am using to hack together support for a `get` method on the parameter object.  I originally had overridden each of the Parameter types we use from `flask_marshmallow.base_fields` with a Mix-in for just that method, but found this to be a more extensible solution. 

I wanted this PR to be a minimal drop-in replacement with just the minimum changed.  I propose that a refactoring of the codebase in relation to updating (and hopefully slimming down) `flask_restx_patched` will come in the future.  This implementation of `expected_params` is to get the ball rolling with a working migration.  I think this [line](https://github.com/WildMeOrg/houston/blob/master/flask_restplus_patched/namespace.py#L92) will offer some evidence on how to eliminate the need for this patched function.

Reply:
By the way, I'm not entirely convinced this patch is the correct path to take.  There seems to be upstream state being set by Swagger and the implementation of `extract_resource_doc` between `Flask-RESTPlus` and `Flask-RESTX` are identical in their expectations of a `get` function.  After tearing into this for a week, and about three alternatives, this attempts to squash the downstream state issue and gets us moving forward until a more wholistic audit of what is actually needed in `flask_restx_patched` can be performed.